### PR TITLE
Quote controller first implementation

### PR DESCRIPTION
**Pull request by shaggy8871**
For now, run as http://uselessapi.com/quote?text=You%20suck

### CODE REVIEW

**Comment by dfourn** [comment]
Looks good

File: app/Http/Controllers/QuoteController.php [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+class QuoteController extends Controller
+{
+    const QUOTE_CHAR_L = '“';
+    const QUOTE_CHAR_R = '”';
+    const DASH_CHAR = ' — ';
+
+    /**
+     * @return Response
+     */
+    public function parse(Request $request)
```
Comment (mikeaag) [inline app/Http/Controllers/QuoteController.php]: Can we change the method name to 'index' to match the RESTful naming system - https://laravel.com/docs/5.1/controllers#restful-resource-controllers

Comment (shaggy8871) [inline app/Http/Controllers/QuoteController.php]: no

Comment (dfourn) [inline app/Http/Controllers/QuoteController.php]: I can do that on my phone even

File: app/Http/routes.php [hunk 2]
```diff
@@ -29,3 +29,20 @@
...
+
+/*
+|--------------------------------------------------------------------------
+| Useless Routes
+|--------------------------------------------------------------------------
+*/
+
+Route::get('quote', 'QuoteController@parse');
+
+/*
```
Comment (mikeaag) [inline app/Http/routes.php]: Can we add a message to this comment to state what it does :D

**Comment by mikeaag** [comment]
@shaggy8871 Any chance of getting some tests for this :P

**Pull request merged**